src/main/java/seedu/address/model/student/CourseContainsKeywordsPredicate.java

```diff
@@ -0,0 +1,33 @@
+package seedu.address.model.student;
+
+import java.util.List;
+import java.util.function.Predicate;
+
+import seedu.address.commons.util.StringUtil;
+
+/**
+ * Tests that a {@code Course}'s {@code courseCode} matches any of the keywords given.
```

Comment:
I think we want course matching to be exact and case-insensitive, because a tutor for CS2040 may want to exclude students from CS2040S for example.

Reply:
I've worked on this, see #71.